### DISCUSSION TITLE
An image info section?

### DISCUSSION THREAD

**Discussion by Lolcupcakes** [jordenyt/stable_diffusion_sketch#22] [Ideas]
I'd love to be able to open an image and review the details like what model was used, prompt, sampler and such. Is that possible? Thanks again for your hard work on this amazing app!

**Comment by jordenyt** [comment]
Did you try open image generate newly by this app with "PNG info" tab in SD-webui? Can you find the info you want?

**Reply by Lolcupcakes** [reply]
That does work! Although if I'm away from my computer it's not the easiest to do is all. Thanks ^^

**Reply by jordenyt** [reply]
I think you can access that page in your Android device browser. No need to get back to your PC.
